### PR TITLE
Distribute License file in the released tarball - Fixes #65

### LINKED ISSUE #65: License file is not included in release tarball on PyPi
Include license file for those, who consume the release from PyPI.
apypie dependencies do it:
```
$ find ~/.virtualenvs/apypie/ -type f -name "*LICENSE*" 
/home/ogajduse/.virtualenvs/apypie/lib/python3.7/site-packages/wheel-0.34.2.dist-info/LICENSE.txt
/home/ogajduse/.virtualenvs/apypie/lib/python3.7/site-packages/setuptools-46.1.3.dist-info/LICENSE
/home/ogajduse/.virtualenvs/apypie/lib/python3.7/site-packages/urllib3-1.25.8.dist-info/LICENSE.txt
/home/ogajduse/.virtualenvs/apypie/lib/python3.7/site-packages/pip-20.0.2.dist-info/LICENSE.txt
/home/ogajduse/.virtualenvs/apypie/lib/python3.7/site-packages/certifi-2020.4.5.dist-info/LICENSE
/home/ogajduse/.virtualenvs/apypie/lib/python3.7/site-packages/idna-2.9.dist-info/LICENSE.rst
/home/ogajduse/.virtualenvs/apypie/lib/python3.7/site-packages/requests-2.23.0.dist-info/LICENSE
```

### PR DESCRIPTION
**Pull request by ogajduse**

### CODE REVIEW

**Comment by ogajduse** [comment]
```
$ python setup.py sdist > /dev/null && tar -tf dist/apypie-0.2.1.tar.gz | grep LICENSE
apypie-0.2.1/LICENSE
```

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[metadata]
+license_file=LICENSE
+
```
Comment (ekohl) [inline setup.cfg:3]: Nit: is this empty line needed?

Comment (ogajduse) [reply setup.cfg:3]: not at all, will delete

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**